test/apparmor.bats

```diff
@@ -110,7 +111,29 @@ run_a_container_with_wrong_apparmor_profile_name() {
 	cleanup_test
 }
 
-# 5. test running with default apparmor profile unloaded.
+# 5. test running with default apparmor profile new field used.
+# test that we will fail when running a ctr with wrong apparmor profile name.
+run_a_container_after_unloading_default_apparmor_profile_new_field() {
+	local output status
+
+	load_apparmor_profile "$FAKE_CRIO_DEFAULT_PROFILE_PATH"
+	setup_test
+	start_crio "$FAKE_CRIO_DEFAULT_PROFILE_NAME"
+	remove_apparmor_profile "$FAKE_CRIO_DEFAULT_PROFILE_PATH"
+
+	jq '	  .linux.security_context.apparmor.profile_type = 0 | .linux.security_context.apparmor.localhost_ref = "runtime/default"' \
```

Comment:
why the large whitespace for jq?

Reply:
> why the large whitespace for jq?

idk, mayber copy-pasta 😄

Reply:
yep 

https://github.com/cri-o/cri-o/blob/93b05e56d222628d18e7a79fd7a8761323460bee/test/apparmor.bats#L123-L126

Reply:
if we don't need it I'd say drop it.

Reply:
you can follow up if you want.

Reply:
it's pretty standard. it was originally suggested by @kolyshkin to line up the command in an easier to read way